### DISCUSSION TITLE
Missing HTML links

### DISCUSSION THREAD

**Discussion by Bmcdaid** [jgraph/drawio#3548] [General]
I am no longer able to access my previous drawings. I have tried to access three drawings e.g. Design Process 11222 (where the logo image has changed) 

![Html links](https://user-images.githubusercontent.com/132032055/235042186-839decff-6414-4c60-b7db-7d62abec291e.JPG)

Has there been an update that has lead to these being deleted? 
If not how can I access them?
Thanks

**Comment by davidjgraph** [comment]
Where did you store them?

**Comment by Bmcdaid** [comment]
Hi David,
I have stored them as a htlm online.

***@***.***

Now when I open the link
https://www.diagrameditor.com/webapp/?ui=atlas&tr=0&gh=0&gl=0&gapi=0&od=0&db=0&lang=en#LDesign%20Process%20230822.drawio

It says file not found.

***@***.***

Thanks,
Brendan

**Comment by davidjgraph** [comment]
diagrameditor.com is not our site. If someone has taken and hosted our project, they are responsible for it.
